src/TaskManager/TaskManager/appsettings.Development.json

```diff
@@ -27,9 +28,9 @@
       "serviceAssemblyName": "Monai.Deploy.Storage.MinIO.MinIoStorageService, Monai.Deploy.Storage.MinIO",
       "settings": {
         "endpoint": "localhost:9000",
-        "accessKey": "minioadmin",
```

Comment:
Think changing this here will break the mwm-developer-setup.md walkthough, the argo setup sets the user/pass on minio as minioadmin / minioadmin

Reply:
Reverted